bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,94 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
+    extract_documentation)
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class DocumentationStyleBear(LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/7sfk3i9oxs1ixg2ncsu3pym0u'
+    CAN_DETECT = {'Documentation'}
+
+    def run(self, filename, file, docstyle: str, language: str):
+        """
+        Checks for certain in-code documentation styles.
+
+        It currently checks for the following styles:
+
+        hello
+        :param x:
+            4 space indent
+        :return:
+            also 4 space indent
+            following lines are also 4 space indented
+        """
+        for doc_comment in extract_documentation(file, language, docstyle):
+
+            metadata = iter(doc_comment.parse())
+
+            # Assuming that the first element is always the only description.
+            description = next(metadata)
+
+            new_metadata = []
+            for m in metadata:
+                stripped_desc = map(str.lstrip, m.desc.splitlines(True))
+                new_desc = ""
+                for line in stripped_desc:
+                    new_desc += (("\n" if not line.endswith("\n") else "") +
+                                 (" " * 4 if line != "" else "") + line)
+                new_metadata.append(m._replace(desc=new_desc))
+
+            new_metadata = [description] + new_metadata
+
+            new_comment = DocumentationComment.from_metadata(
+                new_metadata, doc_comment.docstyle_definition,
+                doc_comment.marker, doc_comment.indent, doc_comment.range)
+
+            def occurrences(string, sub):
+                count = start = 0
+                while True:
+                    start = string.find(sub, start) + 1
+                    if start > 0:
+                        count += 1
+                    else:
+                        return count
+
+            if new_comment != doc_comment:
+                # Something changed, let's apply a result.
+                diff = Diff(file)
+
+                # Delete the old comment
+                for i in range(doc_comment.range.start.line,
+                               doc_comment.range.end.line + 1):
+                    diff.delete_line(i)
```

Comment:
https://github.com/coala-analyzer/coala/pull/2620 is going to come, if it gets merged earlier than yours we could use that :)+